pontoon/sync/changeset.py

```diff
@@ -232,7 +233,15 @@ def bulk_update_entities(self):
             ])
 
     def bulk_create_translations(self):
+        def map_to_translation_memory(translation):
+            return TranslationMemoryEntry(
+                source=translation.entity.string,
+                target=translation.string,
+                locale_id=translation.locale_id
+            )
         Translation.objects.bulk_create(self.translations_to_create)
+        translations = filter(lambda t: t.plural_form in (None, 0), self.translations_to_create)
```

Comment:
Nice! I like this! Even better, map and filter is just what list comprehensions were created for. You could make this more concise using one.